<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>supprimer_equipe</class>
 <widget class="QDialog" name="supprimer_equipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="list_e_nom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>142</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_equipe">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>44</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>104</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Manager de l'équipe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="liste_e_manager">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>103</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>144</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom d'équipe:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste des d'équipes:</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>171</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimer_equipe_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler_la_suppression">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_sup</zorder>
  <zorder>list_e_nom</zorder>
  <zorder>comboBox_equipe</zorder>
  <zorder>label_3</zorder>
  <zorder>liste_e_manager</zorder>
  <zorder>label_7</zorder>
  <zorder>label_10</zorder>
  <zorder>listView</zorder>
  <zorder>image_2</zorder>
  <zorder>image_3</zorder>
  <zorder>supprimer_equipe_2</zorder>
  <zorder>annuler_la_suppression</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
